tests/reports/out.xml

```diff
@@ -1 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="0" failures="0" skipped="0" tests="6" time="0.977" timestamp="2022-07-27T09:03:59.457913" hostname="22ce2a36705d"><testcase classname="tests.test_backend" name="test_reconnect_enabled" time="0.440" /><testcase classname="tests.test_backend" name="test_reconnect_disabled" time="0.020" /><testcase classname="tests.test_backend" name="test_reconnect_in_atomic_before_begin[False]" time="0.047" /><testcase classname="tests.test_backend" name="test_reconnect_in_atomic_before_begin[True]" time="0.045" /><testcase classname="tests.test_backend" name="test_not_reconnect_in_transaction[False]" time="0.021" /><testcase classname="tests.test_backend" name="test_not_reconnect_in_transaction[True]" time="0.063" /></testsuite></testsuites>
```

Comment:
Shouldn't this be ignored?

Reply:
Should. I deleted this in this PR and added to gitignore